lib/internal/crypto/random.js

```diff
@@ -119,6 +120,51 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
```

Comment:
I think this limitation should be mentioned in the documentation.

Reply:
It should also be possible to make this work for all integers up to `Number.MIN_SAFE_INTEGER` (53 bits instead of 48).

Reply:
> It should also be possible to make this work for all integers up to `Number.MIN_SAFE_INTEGER` (53 bits instead of 48).

I am using `bytes.readUIntBE(0, 6)` to get the number which has max 48 bits. Is there an alternative API I could use that would read 53 bits of the buffer into a number?

Reply:
I think you would just have to request 7 bytes and do the math to get a 53-bit number manually :woman_shrugging: